components/Pagination/Pagination.tsx

```diff
@@ -1,43 +1,54 @@
-import React, { useEffect } from 'react';
-import { useTheme } from 'next-themes';
-import clsx from 'clsx';
+import React, { useEffect } from 'react'
+import { useTheme } from 'next-themes'
+import clsx from 'clsx'
 
 type PaginationProps = {
-  totalPages: number[] | null;
-  currentPage: number;
-  handlePageChange: (page: number) => void;
-};
+  toporbottom:boolean
+  totalPages: number[] | null
+  currentPage: number
+  handlePageChange: (page: number) => void
+}
 
 export default function Pagination({
+  toporbottom,
   totalPages,
   currentPage,
   handlePageChange,
 }: PaginationProps) {
-  const { resolvedTheme } = useTheme();
-  const isDarkMode = resolvedTheme === 'dark';
-
+  const { resolvedTheme } = useTheme()
+  const isDarkMode = resolvedTheme === 'dark'
   const scrollToTop = () => {
-    window.scrollTo({ top: 0, behavior: 'smooth' });
-  };
+    window.scrollTo({ top: 0, behavior: 'smooth' })
+  }
 
   const changePage = (page: number) => {
-    handlePageChange(page);
-  };
+    handlePageChange(page)
+  }
 
   useEffect(() => {
-    scrollToTop();
-  }, [currentPage]);
+    scrollToTop()
+  }, [currentPage])
 
   return (
     <>
       {totalPages && totalPages.length > 1 && (
         <div
-          className={clsx(
-            'w-full z-20 flex lg:w-full items-center justify-center ',
-            'absolute bottom-2 right-0'
-          )}
+          className={
+            toporbottom == true
+              ? clsx(
+                  'w-full z-20 flex lg:w-full items-center justify-center ',
+                  'absolute bottom-2 right-0'
+                )
+              : clsx(
+                  ' z-20 flex w-full lg:w-full items-center justify-end ',
+                  'absolute top-0 right-0 '
+                )
+          }
```

Comment:
I think we can achieve this in a single `clsx` wrapper.

```suggestion
          className={clsx(
            toporbottom == true
              ?  'w-full z-20 flex lg:w-full items-center justify-center absolute bottom-2 right-0'
              : 'z-20 flex w-full lg:w-full items-center justify-end absolute top-0 right-0'
           )}
```